plugins/nextcloud/NextcloudAddressBook.php

```diff
@@ -0,0 +1,96 @@
+<?php
+use RainLoop\Providers\AddressBook\Classes\Contact;
+use Sabre\VObject\Component\VCard;
+
+class NextcloudAddressBook implements \RainLoop\Providers\AddressBook\AddressBookInterface
+{
+	use Rainloop\Providers\AddressBook\CardDAV;
+	private const URI = 'webmail';
```

Comment:
This can probably be a default and we can probably allow to set via config.

Also this addressbook must exist..